scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/ForkedTaskVariablesManager.java

```diff
@@ -65,9 +65,10 @@
     }
 
     public void addBindingsToScriptHandler(ScriptHandler scriptHandler, TaskContext taskContext, VariablesMap variables,
-            Map<String, String> thirdPartyCredentials, SchedulerNodeClient client, RemoteSpace userSpaceClient,
-            RemoteSpace globalSpaceClient, Map<String, String> resultMetadata) {
+            Map<String, Serializable> JobMap, Map<String, String> thirdPartyCredentials, SchedulerNodeClient client,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)